spec/venue_spec.rb

```diff
@@ -44,12 +44,40 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+  describe '#over_capacity' do
+    it 'returns true if more patrons than capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.over_capacity?
+      expect(venue.over_capacity?).to eq false
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      venue.over_capacity?
+      expect(venue.over_capacity?).to eq true
+    end
+  end
+  describe '#kick_out' do
+    it 'removes patrons to become at capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      venue.kick_out
+      expect(venue.number_of_patrons).to eq 4
```

Comment:
This is not doing what you are expecting it to do. Your `number_of_patrons` method is returning 5.